certified-connectors/Evocom/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2416 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "Evocom",
+    "description": "Evocom is an easy-to-use solution for collaborative workflows and task management for enterprises. According to the motto \"first digitize, then automate\", companies transform manual processes into digital processes in order to further integrate and automate them afterwards.",
```

Comment:
The backslashes won't be parsed when the description is shown in-product, so they can be removed, leaving simply `to the motto "first digitize, then automate"`

Reply:
How would this work without the backslashes (escaping double quotes)? There are already double quotes at the beginning and at the end of the string.

Reply:
Hey @denis-sisic - that was actually a mistype on my part, I meant to offer single quotes, so `'first digitize, then automate'`. Apologies! Thanks for flagging.

However, your question got me thinking, so I did some digging around. It does look like the escape characters are respected, so you should be able to leverage them as-is. My apologies on the confusion on that side as well ;)